assets/shaders/octree/_coneTrace.glsl

```diff
@@ -100,9 +100,14 @@ vec4 coneTrace(
                 node,
                 parentNode
             ); // TODO: We are visiting the same node twice for some reason
+
+            if (changedOctreeLevel) {
+                // To account for the larger voxelSize in the new level
+                //sampleStep *= 2; // Increase sampleStep, same as increasing voxelSize by 2
+                sampleStep = (1 / pow(2, octreeLevel + 1));
+            }
```

Comment:
In case we skip an octree level from one step to another, not sure if it is posible, but it is more resilient to unexpected scenarios than multiplying `*2`